geowebcache/core/src/test/java/org/geowebcache/seed/TruncateBboxRequestTest.java

```diff
@@ -58,7 +60,20 @@ protected SeedRequest seedRequest(
                         hasProperty("filterUpdate", any(Boolean.class)),
                         hasProperty("threadCount", any(Integer.class)),
                         hasProperty("bounds", equalTo(bounds)),
-                        hasProperty("type", any(GWCTask.TYPE.class))));
+                        hasProperty("type", any(GWCTask.TYPE.class)));
+        EasyMock.reportMatcher(
+                new IArgumentMatcher() {
+
+                    @Override
+                    public boolean matches(Object argument) {
+                        return matcher.matches(argument);
+                    }
+
+                    @Override
+                    public void appendTo(StringBuffer buffer) {
+                        matcher.describeTo(new StringDescription(buffer));
+                    }
+                });
         return null;
```

Comment:
This bit has me confused.. if we return null, is the object being built in the previous step even used?

Reply:
The old and new code both return null here. I think that easymock is adding the matcher to a static field so the return value isn't very important. I personally find mockito to be clearer and easier to use even though geoserver uses easymock more.